presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1386,9 +1385,9 @@ private void printPlanNodesStatsAndCost(int indent, PlanNode... nodes)
 
         private Optional<String> formatPlanNodeStatsAndCost(PlanNode node)
         {
-            PlanNodeStatsEstimate stats = estimatedStatsAndCosts.getStats().getOrDefault(node.getId(), UNKNOWN_STATS);
+            PlanNodeStatsEstimate stats = estimatedStatsAndCosts.getStats().getOrDefault(node.getId(), PlanNodeStatsEstimate.unknown());
             PlanNodeCostEstimate cost = estimatedStatsAndCosts.getCosts().getOrDefault(node.getId(), UNKNOWN_COST);
-            if (stats.equals(UNKNOWN_STATS) || cost.equals(UNKNOWN_COST)) {
+            if (stats.isOutputRowCountUnknown() || cost.equals(UNKNOWN_COST)) {
```

Comment:
The change here does not belong to "Add PlanNodeStatsEstimate#unknown() method" commit. This commit should be a plain refactor.

Reply:
> This commit should be a plain refactor.

Do you mean it should go as a separate commit?